app/controllers/v0/messages_controller.rb

```diff
@@ -26,8 +26,12 @@ def show
     def create
       message = Message.new(message_params)
       raise Common::Exceptions::ValidationErrors, message unless message.valid?
-
-      response = client.post_create_message(message_params)
+      response =
+        if request.content_type == 'multipart/form-data'
```

Comment:
My plan is to go in and require all endpoints to require the data, type, id (if applicable), and in this case it could be a compound document with attachment. But I think checking that it is multipart is more descriptive than checking for presence of file, in that you can not provide a link to a file, it must come in as multi-part. If we were doing base64 encoding / decoding and including in the JSON hash then it might make more sense?